2.x-activity-module/mod/aspirelists/launch.php

```diff
@@ -15,8 +18,17 @@
 $cm = get_coursemodule_from_id('aspirelists', $id, 0, false, MUST_EXIST);
 $list = $DB->get_record('aspirelists', array('id' => $cm->instance), '*', MUST_EXIST);
 $course = $DB->get_record('course', array('id' => $cm->course), '*', MUST_EXIST);
+$context = context_module::instance($cm->id);
 
-add_to_log($course->id, "aspirelists", "launch", "launch.php?id=$cm->id", "$list->id");
```

Comment:
Which versions of moodle are going to lose the ability to log with this line removed?